app/views/content/train-to-be-a-teacher/teaching-internships.md

```diff
@@ -10,23 +10,14 @@ promo_content:
   - content/train-to-be-a-teacher/promos/eta-promo-internships
 backlink: /
 fullwidth: true
-navigation: 20.30
+navigation: 20.25
```

Comment:
@sarahcrack @martyn-w I think the script needs updating slightly as this change isn't needed

Emma changed this nav order from `20.25` to `20.30` when she created the new placements page as part of https://github.com/DFE-Digital/get-into-teaching-app/pull/4213 so we need to keep it at `20.30`

Reply:
(completely forgot about the internship script when approving Emma's PR!)